src/bp/ui-shared/src/ConfirmDialog/style.scss

```diff
@@ -1,3 +1,14 @@
-.ConfirmDialogContainer > div{
+.ConfirmDialogContainer > div {
   z-index: 9999;
+
+  :global(.bp3-button-text) {
+    font-size: 12px !important;
+  }
+}
+
+.icon {
+  display: block !important;
```

Comment:
no need for display: block if you have float: left, unless there's something that need to be overwritten